### DISCUSSION TITLE
跳转白屏

### DISCUSSION THREAD

**Discussion by oeyoews** [tangly1024/NotionNext#587] [General]
页面直接跳转总会有短暂的白屏情况出现正常吗

**Comment by tangly1024** [comment]
不太正常哦， 能具体描述一下操作的步骤吗

**Reply by oeyoews** [reply]
@tangly1024 就是在打开一篇文章后（ios)， 右滑返回博客主页（直接点击home正常)会出现短暂白屏， 像是又重新刷新了一次

**Reply by tangly1024** [reply]
确实如此，有短暂的白屏。

这个白屏应该是动画效果导致，主页的卡牌列表有添加`淡入淡出`的效果，在移动端左滑时会重新播放此动画。
